CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Fixed `yarn add @algo-builder/web` (was failing because of missing dependency `zod` in packages/web).
 * Fix metadatahash type
 * Fix init project-name bug(`algob init <project-name>` command was not working properly)
+* Fix random address for logicsig(runtime)
```

Comment:
Could you please describe the bug?

Reply:
The bug is that we need a predictable address of an lsig. Creating two times an lsig account from the same TEAL code should return the same address.